index.js

```diff
@@ -1,7 +1,7 @@
-const DirectorySwitcherPlugin = require('./config/webpack/plugin/resolve/DirectorySwitcherPlugin');
-const LocalPackageAliasPlugin = require('./config/webpack/plugin/resolve/LocalPackageAliasPlugin');
-const TerraDevSiteEntrypoints = require('./config/webpack/plugin/TerraDevSiteEntrypoints');
-const TerraDevSite = require('./config/webpack/plugin/TerraDevSite');
+const DirectorySwitcherPlugin = require('./lib/webpack/plugin/resolve/DirectorySwitcherPlugin');
```

Comment:
all files have been consolidated in source and for webpack files, ignored and copied to lib.

Reply:
Should these be scoped under a config file still? That'd align with terra-functional-testing: https://github.com/cerner/terra-toolkit/tree/main/packages/terra-functional-testing/src/config

Reply:
They aren't a shared config file like the wdio config. This houses the webpack plugins so i think they're scoped appropriately.